packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -529,6 +530,9 @@ class ScrollableBaseNative extends Component {
 	// scroll animator
 	animator = new ScrollAnimator()
 
+	// platform info
+	platformInfo = platform
```

Comment:
Is there a reason to make a copy of platform?

Reply:
I deleted this copy of platform. The platform did not need to be copied.